js/constants/appConfig.js

```diff
@@ -119,6 +119,9 @@ module.exports = {
     'bookmarks.toolbar.showOnlyFavicon': false,
     'payments.enabled': false,
     'payments.notifications': false,
+    // "Out of money, pls add" / "In 24h we'll pay publishers [Review]"
+    // After shown, set timestamp to next reconcile time - 1 day.
+    'payments.notification-reconcile-soon-timestamp': null,
     'payments.notificationTryPaymentsDismissed': false, // True if you dismiss the message or enable Payments
```

Comment:
This might be more of a @bradleyrichter question... but I wonder if this should this be covering both cases? Let's say you're within 24 hours of reconciling and you only have the ~5 dollars in your account. You'd dismiss the nag window but then you're out of budget.  This is a case where you might want to get nagged again sometime before the next reconciliation period (especially since getting funds in might take > 24 hours)